requirements/common.in

```diff
@@ -28,7 +28,12 @@ SQLAlchemy==1.4.*
 
 # Needed for S3 file uploads and other AWS tools
 boto3
-boto3-stubs[ses,s3,sqs,sns]
+
+# The runtime-relevant part of boto3-stubs (see mypy.in)
+mypy-boto3-s3
+mypy-boto3-ses
+mypy-boto3-sns
+mypy-boto3-sqs
```

Comment:
Is `tools/tail-ses` intended to be run in a production environment? Its location in `tools/` indicates that it’s not, in which case we could exclude these packages too.

Reply:
I think it's intended to be run in a production environment installed via `upgrade-zulip-from-git`, where `tools` is available... but it might be more appropriate to move it a new `scripts/aws` or something like it.

Reply:
It's intended to be run in production, or maybe only-some-production, or possibly non-http-production hosts which are trusted with AWS credentials.

I think it's reasonable to think about splitting out some probably-only-our-deployment-relevant scripts and dependencies into their own area so we don't pollute the main deps.